### PR TITLE
Filter Books by Category added

### PR DESCRIPTION
**Pull request by Nikita5796**

### CODE REVIEW

File: src/app/book/booklist/booklist.component.ts [hunk 1]
```diff
@@ -10,6 +10,9 @@ import { NgbRatingConfig } from '@ng-bootstrap/ng-bootstrap';
 })
 export class BooklistComponent implements OnInit {
 
+  BookCategories = {};
```
Comment (bala0105) [inline src/app/book/booklist/booklist.component.ts]: variable name should be bookCategories. :(

File: src/app/book/booklist/booklist.component.html [hunk 2]
```diff
@@ -1,20 +1,30 @@
...
-      <td>{{ book.price | currency: "INR" }}</td>
-      <td><ngb-rating [rate] = "book.ratings"></ngb-rating></td>
-    </tr>
-  </tbody>
-</table>
+<div class = "container">
+  <div class = "row">
+    <select class="form-control" name="categoryId" [(ngModel)]="categoryId" (change)="categoryBookList(categoryId)" >
+      <option value="0">--All Books--</option>
+      <option *ngFor="let category of BookCategories" value="{{ category.id }}">{{
```
Comment (bala0105) [inline src/app/book/booklist/booklist.component.html]: variable name should be bookCategories. :(

**Review by bala0105 — CHANGES_REQUESTED** [review]

File: src/app/book/booklist/booklist.component.ts (hunk 1 above)
Comment (Nikita5796) [reply src/app/book/booklist/booklist.component.ts]: done.

File: src/app/book/booklist/booklist.component.html (hunk 2 above)
Comment (Nikita5796) [reply src/app/book/booklist/booklist.component.html]: done.

**Review by ketannaik — APPROVED** [review]

**Review by bala0105 — APPROVED** [review]

**Pull request merged**